otherlibs/dune-rpc/private/procedures.ml

```diff
@@ -52,7 +52,10 @@ module Public = struct
   end
 
   module Promote = struct
-    let v1 = Decl.Request.make_current_gen ~req:Path.sexp ~resp:Conv.unit ~version:1
+    let v1 =
+      Decl.Request.make_current_gen ~req:(Conv.list Path.sexp) ~resp:Conv.unit ~version:1
```

Comment:
I think you either need to introduce a new generation of this RPC or add a separate RPC. Existing RPC shouldn't be modified like this.